packages/twenty-front/.env.example

```diff
@@ -3,6 +3,12 @@ GENERATE_SOURCEMAP=false
 
 # ———————— Optional ————————
 # REACT_APP_PORT=3001
+# REACT_APP_BASE_URL=http://localhost
 # CHROMATIC_PROJECT_TOKEN=
 # VITE_DISABLE_TYPESCRIPT_CHECKER=true
-# VITE_DISABLE_ESLINT_CHECKER=true
\ No newline at end of file
+# VITE_DISABLE_ESLINT_CHECKER=true
+
+######  --------------> !!! FOR CHARLES AND FELIX !!! we can create a gist in twenty if you want <---------------------------
+###### To configure a local certificate you can follow these step https://gist.github.com/AMoreaux/635ca9c38924d42a4d914dabe4376f72
```

Comment:
Ideally we would add this to the developer documentation (mdx file) but I think it fits well with your gist like that for now. Could you just remove your comment so that we can merge?

Reply:
We choose to change that in this PR / commit:
https://github.com/twentyhq/twenty/pull/8579/commits/8cf65020d1a33e7ca3120ae44f6e3abab7caa570

So now it should be good.